csrc/flash_attn/flash_api.cpp

```diff
@@ -255,18 +262,18 @@ void set_params_alibi(Flash_fwd_params &params, c10::optional<at::Tensor> &alibi
 #endif
 }
 
-std::vector<at::Tensor>
+std::array<at::Tensor, 8>
```

Comment:
Why change this to an array?

Reply:
If I remember correctly, it didn't properly work with torch extension but I can try again

Reply:
`std::vector<at::Tensor>` should work.  The schema type would need to be `Tensor[]`.